### PR TITLE
FDN-112: Add jira/valid-reference label for all repos

### PR DESCRIPTION
**Pull request by bparees**
intent is to make this label available for use on all repos, for use as a future PR merge gating mechanism

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/test-platform ptal

File: core-services/prow/02_config/_labels.yaml [hunk 1]
```diff
@@ -369,6 +369,12 @@ default:
       target: prs
       prowPlugin: backport-verifier
       addedBy: prow
+    - color: 15dd18
+      description: Indicates that this PR references a valid Jira ticket of any type.
```
Comment (wking) [inline core-services/prow/02_config/_labels.yaml:373]: It sounds like pulls referencing a valid Jira bug will have both `jira/valid-bug` and `jira/valid-reference`?  Possibly worth pointing that out as an explicit choice in the commit message, so folks don't wonder about trying to remove overlap later.

Comment (bparees) [reply core-services/prow/02_config/_labels.yaml:373]: > It sounds like pulls referencing a valid Jira bug will have both jira/valid-bug and jira/valid-reference

yes.

i'll fix up the commit message while also changing this to a unique color (forgot to do that)

Comment (bparees) [reply core-services/prow/02_config/_labels.yaml:373]: actually maybe the color (Green) is as it should be.

Comment (bparees) [reply core-services/prow/02_config/_labels.yaml:373]: commit text updated

**Review by droslean — APPROVED** [review]

**Pull request merged**